.pre-commit-config.yaml

```diff
@@ -7,4 +7,4 @@ repos:
   - repo: https://github.com/shellcheck-py/shellcheck-py
     rev: v0.7.1.1
     hooks:
-      - id: shellcheck
+      - id: shellcheck
```

Comment:
Revert :)

Reply:
May you explain do you mean?

Reply:
For some reason it doesn't show it here but the only difference between these lines is that new one doesn't have new line symbol at the end. This change is completely unnecessary.